Flow.Launcher.Infrastructure/StringMatcher.cs

```diff
@@ -126,27 +177,39 @@ public MatchResult FuzzyMatch(string query, string stringToCompare, MatchOption
                 if (currentQuerySubstringCharacterIndex == currentQuerySubstring.Length)
                 {
                     // if any of the substrings was not matched then consider as all are not matched
-                    allSubstringsContainedInCompareString = matchFoundInPreviousLoop && allSubstringsContainedInCompareString;
+                    allSubstringsContainedInCompareString =
+                        matchFoundInPreviousLoop && allSubstringsContainedInCompareString;
 
                     currentQuerySubstringIndex++;
 
-                    allQuerySubstringsMatched = AllQuerySubstringsMatched(currentQuerySubstringIndex, querySubstrings.Length);
+                    allQuerySubstringsMatched =
+                        AllQuerySubstringsMatched(currentQuerySubstringIndex, querySubstrings.Length);
+
                     if (allQuerySubstringsMatched)
-                        break;
+                        continue;
```

Comment:
this change is correct?

Reply:
Em yes, that is used to before early return of full match to check more acronym.